### DISCUSSION TITLE
Type assertion , MerkleAirDrop ..! Need a quick help!

### DISCUSSION THREAD

**Discussion by AllenGeorge08** [Cyfrin/foundry-full-course-cu#2025] [Q&A]
I byte32 asserted the proofs here. Should i add a abi.encodedLayer  too?  Would the second one be better?

I mean in the lecture the proofs weren't bytes32 encoded, but in my vscode it was throwing an error saying that it was uint256 , so i asserted it...Suggestions?

` bytes32[] public PROOF = [bytes32(0x0fd7c981d39bece61f7499702bf59b3114a90e66b51ba2c53abdf7b62986c00a),bytes32(0xe5ebd1e1b5a5478a944ecab36a9a954ac3b6b8216875f6524caa7a1d87096576)];
`

**Comment by EngrPips** [comment]
Keep crushing it.
